### PR TITLE
perlPackages.CLIHelpers: init at 1.8

### PR DESCRIPTION
**Pull request by invokesus**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by aanderse** [comment]
@GrahamcOfBorg build perlPackages.CLIHelpers perldevelPackages.CLIHelpers

**Comment by invokesus** [comment]
Can you rerun ofborg? Not sure why it failed. I'm able to build `perlPackages.PerlCritic` on `nixpkgs-unstable`.

**Comment by aanderse** [comment]
@GrahamcOfBorg build perlPackages.CLIHelpers perldevelPackages.CLIHelpers

File: pkgs/top-level/perl-packages.nix [hunk 1]
```diff
@@ -2754,6 +2754,22 @@ let
     };
   };
 
+  CLIHelpers = buildPerlPackage {
+    pname = "CLI-Helpers";
+    version = "1.6";
```
Comment (stigtsp) [inline pkgs/top-level/perl-packages.nix:2759]:
```suggestion
    version = "1.8";
```

File: pkgs/top-level/perl-packages.nix [hunk 2]
```diff
@@ -2754,6 +2754,22 @@ let
...
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/B/BL/BLHOTSKY/CLI-Helpers-1.6.tar.gz";
+      sha256 = "5f2e107c435220e714dae35d5e7d15ca9a51fac5ceb9fb28c84dee1770955ee2";
```
Comment (stigtsp) [inline pkgs/top-level/perl-packages.nix:2762]:
```suggestion
      sha256 = "1hgiynpy7q4gbx1d9pwnzdzil36k13vjxhscalj710ikcddvjz92";
```

**Review by stigtsp — COMMENTED** [review]
1.8 is the latest version on CPAN.

Other than that, looks good to me.

File: pkgs/top-level/perl-packages.nix [hunk 3]
```diff
@@ -2754,6 +2754,22 @@ let
     };
   };
 
+  CLIHelpers = buildPerlPackage {
+    pname = "CLI-Helpers";
+    version = "1.6";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/B/BL/BLHOTSKY/CLI-Helpers-1.6.tar.gz";
```
Comment (stigtsp) [inline pkgs/top-level/perl-packages.nix:2761]:
```suggestion
      url = "mirror://cpan/authors/id/B/BL/BLHOTSKY/CLI-Helpers-1.8.tar.gz";
```

**Comment by invokesus** [comment]
Thanks for the review @stigtsp. Amended and force-pushed. :)

**Review by stigtsp — CHANGES_REQUESTED** [review]
Hi @invokesus thx for looking at the PRs again. I've tested your changes on Darwin and Linux and it looks good to me.

Just one small nitpick. The commit message says `perlPackage.`, it should say `perlPackages.` (plural)

**Comment by invokesus** [comment]
Thanks for catching that. Fixed now.

**Review by stigtsp — APPROVED** [review]

**Comment by stigtsp** [comment]
@invokesus Thx for the contribution!

**Pull request merged**